### ISSUE TITLE
Replace `ComputeConsumerUnbondingPeriod` with a parameter.

### ISSUE THREAD

**Issue by jtremback** [cosmos/interchain-security#340]
There's no reason to compute the consumer's unbonding period on the fly. Provider validators should inspect a consumer's `genesis.json` to see if the unbonding period is satisfactory. Make a parameter and replace all uses of `ComputeConsumerUnbondingPeriod` with that parameter.

**Comment by mpoke** [comment]
If we make the `ConsumerUnbondingPeriod` a param, we need to make sure that the consumer cannot change it without the approval of the provider validators. 

Also, the value of `ConsumerUnbondingPeriod` affects unbonding (at least now). The validators need to be educated on the impact  of this param.

**Comment by jtremback** [comment]
Yea, there are already a lot of other things that the consumer shouldn't be able to change, and a core part of the democracy work that Stana and Dusan are doing is to whitelist the safe ones. By default it can't be changed.

**Comment by smarshall-spitzbart** [comment]
I'll add that I'll have to change the default unbonding period for the ccv provider to satisfy checks [like this](https://github.com/cosmos/interchain-security/blob/main/tests/e2e/common_test.go#L220) where provider/consumer unbonding are compared directly. E2e tests do not have a way to easily set an unbonding period for a chain, so the staking module default unbonding period is always used at e2e test runtime. The runtime value needs to align with the ccv defaults.

TLDR: [this](https://github.com/cosmos/cosmos-sdk/blob/main/x/staking/types/params.go#L21) needs to be equal to [this](https://github.com/cosmos/interchain-security/blob/main/x/ccv/provider/types/params.go#L19) for e2e tests assertions to pass, due to lack of customizability in those tests. If it's unacceptable to change the unbonding period defaults in the ccv module, we'll need to talk about alternatives
